### PR TITLE
Add instance_group to bulk api

### PR DESCRIPTION
**Pull request by gamuniz**
##### SUMMARY

Added instance group code back into bulk job serializer after merging IG role code.

##### ISSUE TYPE

 - New or Enhanced Feature

##### COMPONENT NAME

 - API
 

##### AWX VERSION

```
make VERSION 
awx: 22.1.1.dev83+gdd2b3cabdd.d20230511

```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Comment by gamuniz** [comment]
related to #13981

File: awx/api/serializers.py [hunk 1]
```diff
@@ -4687,11 +4687,11 @@ class BulkJobNodeSerializer(WorkflowJobNodeSerializer):
     credentials = serializers.ListField(child=serializers.IntegerField(min_value=1), required=False)
     labels = serializers.ListField(child=serializers.IntegerField(min_value=1), required=False)
     # TODO: Use instance group role added via PR 13584(once merged), for now everything related to instance group is commented
-    # instance_groups = serializers.ListField(child=serializers.IntegerField(min_value=1), required=False)
+    instance_groups = serializers.ListField(child=serializers.IntegerField(min_value=1), required=False)
 
     class Meta:
         model = WorkflowJobNode
-        fields = ('*', 'credentials', 'labels')  # m2m fields are not canonical for WJ nodes, TODO: add instance_groups once supported
+        fields = ('*', 'credentials', 'labels', 'instance_groups')  # m2m fields are not canonical for WJ nodes, TODO: add instance_groups once supported
```
Comment (kdelee) [inline awx/api/serializers.py]:
```suggestion
        fields = ('*', 'credentials', 'labels', 'instance_groups')  # m2m fields are not canonical for WJ nodes
```

**Review by jainnikhil30 — APPROVED** [review]

File: awx/api/serializers.py [hunk 2]
```diff
@@ -4687,11 +4687,11 @@ class BulkJobNodeSerializer(WorkflowJobNodeSerializer):
     credentials = serializers.ListField(child=serializers.IntegerField(min_value=1), required=False)
     labels = serializers.ListField(child=serializers.IntegerField(min_value=1), required=False)
     # TODO: Use instance group role added via PR 13584(once merged), for now everything related to instance group is commented
```
Comment (jay-steurer) [inline awx/api/serializers.py]: Should we take this comment out?

Comment (gamuniz) [reply awx/api/serializers.py]: All set

**Review by jay-steurer — APPROVED** [review]

**Pull request merged**